Add issue templates (#31693)
servo/servo: .github/ISSUE_TEMPLATE/feature_request.md

---

> name: Feature request

Reviewer: ```suggestion
name: Feature Request
```

> about: Suggest an idea or new feature request

Reviewer: ```suggestion
about: Suggest the implementation of a new feature
```

title: ''
labels: B-feature-tracking, C-untriaged
assignees: ''

---

> **Describe the bug:**
> A clear and concise description of the feature request.

Reviewer: ```suggestion
**Describe the new feature:**
A clear and concise description of the new feature with links to any relevant specifications.
```
